egs/ami/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -96,20 +96,20 @@ if [ $stage -le 5 ] && [ "$speed_perturb" == "true" ]; then
 
   mfccdir=mfcc_${mic}_perturbed
   for x in train_sp; do
-    steps/make_mfcc.sh --cmd "$train_cmd" --nj $nj \
+    steps/make_mfcc.sh --cmd "$train_cmd" --nj 10 \
```

Comment:
@freewym better add the nj variable. It makes life easy when you want to modify the number of jobs.